packages/dappmanager/test/integration/migrations/composeMigration.test.int.ts

```diff
@@ -2,103 +2,127 @@ import { expect } from "chai";
 import fs from "fs";
 import { PackageContainer } from "@dappnode/common";
 import { migrateCoreNetworkAndAliasInCompose } from "../../../src/modules/migrations/addAliasToRunningContainers.js";
+import { removeDnsFromPackageComposeFile } from "../../../src/modules/migrations/removeDnsFromComposeFiles.js";
 import { params } from "@dappnode/params";
 import { mockContainer, shellSafe } from "../../testUtils.js";
 
 describe("Migration", () => {
+  const dncoreNetwork = params.DNP_PRIVATE_NETWORK_NAME;
+  const serviceName = "test";
+  const containerName = "DAppNodePackage-test.dnp.dappnode.eth";
+  const randomImage = "chentex/random-logger";
+  const dnpName = "test-migration.dnp.dappnode.eth";
+  const testPackagePath = `${params.REPO_DIR}/test-migration.dnp.dappnode.eth`;
+
   const container: PackageContainer = {
     ...mockContainer,
-    containerName: "DAppNodeCore-dappmanager.dnp.dappnode.eth",
-    dnpName: "test-migration",
-    serviceName: "dappmanager.dnp.dappnode.eth",
+    containerName,
+    dnpName,
+    serviceName,
     networks: [
       { name: "random", ip: "10.0.1.1" },
       { name: "dncore_network", ip: "172.33.1.7" }
     ]
   };
 
+  const composeNoDns = `
+version: '3.5'
+networks:
+  ${dncoreNetwork}:
+    name: ${dncoreNetwork}
+    external: true
+services:
+  ${serviceName}:
+    image: ${randomImage}
+    container_name: ${containerName}
+    restart: always
+    networks:
+      ${dncoreNetwork}:
+        ipv4_address: 172.33.1.7
+        aliases:
+          - ${serviceName}.test-migration.dappnode
+          - ${serviceName}.dappnode`;
+
   const composeAlreadyMigrated = `
```

Comment:
Now that this file has alias migration and dns migration tests, I would rename:
`composeAlreadyMigrated` to `composeAliasesToBeMigrated`
 `composeToBeMigratedBefore` to  `composeAliasesAlreadyMigrated`
 
 Or something like that, so we know that these are used for alias migration and not DNS